fermenter-mda/src/test/java/org/technologybrewery/fermenter/mda/notification/CreateNotificationSteps.java

```diff
@@ -78,6 +79,15 @@ public void a_configuration_that_triggers_a_notification_for_manual_action_with_
         generators.add(generator);
     }
 
+    @Given("^a notification key to suppress$")
+    public void a_notification_key_to_suppress() throws Throwable {
+        MavenSession session = testCase.newMavenSession();
+        Notification notification = new VelocityNotification("test-message-id", new HashSet<>(), "templates/notifications/sample.notification.vm");
+        NotificationCollector.addNotification("test-message-id", notification);
```

Comment:
Q/A: Why not have the generator add the notification so it mirrors the expected usage?